src/harness/testcases/README.md

```diff
@@ -4,3 +4,39 @@
 
 The current test cases are based on the [WinnForum test specification](https://workspace.winnforum.org/higherlogic/ws/public/download/5637/WINNF-TS-0061-V1.1.0%20-%20WG4%20SAS%20Test%20and%20Certification%20Spec.pdf) v1.1 (Jan 2018)
 
+## Security Test Cases
+
+The SCS.11/16, SDS.11/16 and SSS.11/16 TCs just expect the handshake failure and do not validate the reason for failure.<br>
+The correct way to run the fake_sas for these test cases is
+<pre>
+<code>
+<b> $python fake_sas.py --verify_crl --ca ca_crl_chain.cert </b>
```

Comment:
Would it be feasible to start with `--ca ca_crl_chain_sxs11.cert`, and *then* describe the manual alternative?  I imagine that someone trying to run the test just wants a working command, without needing to navigate a bunch of menus.

Put a space after `$`, because that's how a shell is typically formatted.

Reply:
Update   correctly now.